src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -366,6 +366,12 @@ public void setEntry(BibEntry entry) {
 
         // Notify current tab about new entry
         getSelectedTab().notifyAboutFocus(entry);
+        // Notify all the tabs about new entry
```

Comment:
This feels wrong, too. Is this called by the integrity check pop up?

Reply:
Thank you for the comments! 
Yes, this function is called after double clicking an item in the Check Intergrity table. 

Here's my analysis:
After double clicking an item, the functions are called in the following order: 
 `onSelectionChanged()` , `editEntryAndFocusField()` , `setFocusToField()`, `fieldsEditorTab.getShownFields().contains(field)`.

In the function `fieldsEditorTab.getShownFields().contains(field)`, I find that only the Entry Editor tab we have opened once is not null. 

For example, when I first select the item with the non-ascii chars in the field Abstract, the Entry Editor tab ABSTRACT is null :
![image](https://github.com/JabRef/jabref/assets/64955334/c6982b06-e172-4dad-9059-50953ef6b827)

But when I opened the Entry Editor tab ABSTRACT, then go back to the Entry Editor tab Required fields , it's not null:
![image](https://github.com/JabRef/jabref/assets/64955334/eb65050c-ff9d-4d78-9771-cc7d23d0ae64)

That's because the function `getSelectedTab().notifyAboutFocus(entry)` just notify current tab about new entry.